### ISSUE TITLE
Duplicate Content-Disposition: attachment

### ISSUE THREAD

**Issue by mitar** [owncloud/core#21034]
Example response headers on my ownCloud 8.2.1 installation for `/remote.php/webdav/download.mp3`:

```
Cache-Control: no-store, no-cache, must-revalidate, post-check=0, pre-check=0
Connection: keep-alive
Content-Disposition: attachment
Content-Disposition: attachment
Content-Length: 39343336
Content-Security-Policy: default-src 'self'; script-src 'self' 'unsafe-eval'; style-src 'self' 'unsafe-inline'; frame-src *; img-src * data: blob:; font-src 'self' data:; media-src *; connect-src *
Content-Type: audio/mpeg
Date: Tue, 08 Dec 2015 11:11:22 GMT
ETag: "851440c8f536eaebb17a392718dc0e3f"
Expires: Thu, 19 Nov 1981 08:52:00 GMT
Last-Modified: Tue, 08 Dec 2015 10:35:50 GMT
OC-ETag: "851440c8f536eaebb17a392718dc0e3f"
Pragma: no-cache
Server: nginx
X-Content-Type-Options: nosniff
X-Frame-Options: Sameorigin
X-Robots-Tag: none
X-XSS-Protection: 1; mode=block
```

Observe duplicate `Content-Disposition: attachment`.

**Comment by DeepDiver1975** [comment]
@LukasReschke is this us adding that header? THX

**Comment by PVince81** [comment]
Not for me in 8.2.1 or master from git:
```
  % curl -D - -X GET http://root:admin@localhost/owncloud/remote.php/webdav/welcome.txt

HTTP/1.1 200 OK
Date: Tue, 08 Dec 2015 15:54:53 GMT
Server: Apache/2.4.17 (Linux/SUSE)
X-Powered-By: PHP/5.6.15
Set-Cookie: ockbfur1449k=a33ue1r1b4ipl41luh2fn8nddc6d3t42t15nk31p7hcnnc8s5oe1; path=/owncloud; HttpOnly
Expires: Thu, 19 Nov 1981 08:52:00 GMT
Cache-Control: no-store, no-cache, must-revalidate, post-check=0, pre-check=0
Pragma: no-cache
Set-Cookie: oc_sessionPassphrase=xp4M5rin5tTurqq4xvyMzw9nsRLtBHB99xYhz4zBOLNEmVrz7Y%2BkkjzwsOw3EXBirSWyOyyhhaxn8vfR53Cm%2BdV5tpKnNl4SWKWWdrOTtsFIGFEMo8f2rngAayc1xYOT; path=/owncloud; httponly
Content-Security-Policy: default-src 'self'; script-src 'self' 'unsafe-eval'; style-src 'self' 'unsafe-inline'; frame-src *; img-src * data: blob:; font-src 'self' data:; media-src *; connect-src *
X-XSS-Protection: 1; mode=block
X-Content-Type-Options: nosniff
X-Frame-Options: Sameorigin
X-Robots-Tag: none
Set-Cookie: ockbfur1449k=jrgplpc1hc0g6k0lc511mktu599kllncjoi1mnitjj9e83dq37b0; path=/owncloud; HttpOnly
Last-Modified: Tue, 08 Dec 2015 15:54:28 GMT
ETag: "00a02a431b55ec8664314c035b73c7c8"
Content-Length: 163
Content-Disposition: attachment
OC-ETag: "00a02a431b55ec8664314c035b73c7c8"
Content-Type: text/plain;charset=UTF-8

Welcome to your ownCloud account!

This is just an example file for developers and git users. 
The packaged and released versions will come with better examples.
```

I also tried with an mp3 and I get the same correct result.

@mitar do you have any PHP modules installed that might automatically add this header when detecting extensions like mp3?

**Comment by RealRancor** [comment]
Might be also Server: nginx vs Server: Apache/2.4.17 (Linux/SUSE)

**Comment by mitar** [comment]
I am using this [Docker image](https://github.com/cloyne/docker-owncloud).

**Comment by mitar** [comment]
More information:
* I have a Music app installed
* when I go to the Music app to play the mp3, it first does the HEAD request, that HEAD request has duplicate `Content-Disposition`

The problem is with HEAD request. It works even against the default Paris.jpg:

```
curl -I -v 'http:/root:admin@localhost/owncloud/remote.php/webdav/Photos/Paris.jpg' > /dev/null
```

**Comment by PVince81** [comment]
@LukasReschke didn't we fix that recently ?

**Comment by LukasReschke** [comment]
`\OCA\DAV\Connector\Sabre\FilesPlugin::httpGet` somehow called twice. Can reproduce with head…

**Comment by PVince81** [comment]
Still happening on 9.0.4RC1 and 9.1.0RC1

**Comment by PVince81** [comment]
seems fixed on stable10 / 10.0.3
